### PR TITLE
add api vpn access

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: cmd/transfer-pvc/transfer-pvc.go [hunk 1]
```diff
@@ -261,7 +261,7 @@ func (t *TransferPVCOptions) run() error {
 		log.Fatal(err, "error creating stunnel client")
 	}
 
-	s, err = stunnel.GetTransportFromKubeObjects(srcClient, destClient, s.NamespacedNamePair(), e)
+	s, err = stunnel.GetTransportFromKubeObjects(srcClient, destClient, s.NamespacedNamePair(), e, &transport.Options{})
```
Comment (jmontleon) [inline cmd/transfer-pvc/transfer-pvc.go:264]: There is a build error here that is unrelated to my work.

**Review by alaypatel07 — APPROVED** [review]
VISIACK
